docs/src/content/docs/fr/index.mdx

```diff
@@ -39,19 +41,122 @@ import AboutAstro from '~/components/about-astro.astro';
 		Apportez votre langage de balisage préféré. Starlight vous offre une
 		validation frontmatter intégrée avec la sécurité de type TypeScript.
 	</Card>
-	<Card
-		title="Apportez vos propres composants d'interface utilisateur"
-		icon="puzzle"
-	>
+	<Card	title="Apportez vos propres composants d'interface utilisateur" icon="puzzle">
 		Starlight est une solution de documentation complète, indépendante du cadre
 		de travail. Étendez avec React, Vue, Svelte, Solid, et plus encore.
 	</Card>
 </CardGrid>
 
-<AboutAstro title="Présenté par">
-Astro est le framework web tout-en-un conçu pour la vitesse.
-Tirez votre contenu de n'importe où et déployez-le partout, le tout alimenté par vos composants et bibliothèques d'interface utilisateur préférés.
+<TestimonialGrid title="What people are saying">
```

Comment:
This can be safely translated, this is the title of the section.

Reply:
Missed this one. Thanks!